templates/chain.html

```diff
@@ -302,6 +303,36 @@ <h4 id="name"></h4>
             <br>
         </li>
 
+        <div id="planners" class="section-profile planners" style="display: none">
```

Comment:
let's move this section to the 'advanced' section. should be simple enough - just move the navbar entry to the advanced list and then ensure this html block gets a class="advanced" (see the sources block for an example).

this will change the background-color to the "advanced" color, which is a little darker then the normal/campaign sections.